src/main/routes/receiver.ts

```diff
@@ -22,7 +23,13 @@ export default express.Router()
     } else if (useOauth && req.query.code) {
       cookies.set('state', '', { sameSite: 'lax' })
       if (req.query.state !== OAuthHelper.getStateCookie(req)) {
-        throw new Error('Invalid state')
```

Comment:
I wonder if we shouldn't call `appInsights.defaultClient.trackEvent({` AND after that throw exception anyway?

Reply:
No @yarpo, we want to stop throwing an error to stop giving people 500 until we figure out why it is happening.